### DISCUSSION TITLE
PHP FPM not starting?

### DISCUSSION THREAD

**Discussion by miagg** [beyondcode/herd-community#256] [General]
I have a weird issue where Herd will show PHP-FPM as stopped with the following error message.
  
![Screenshot 2023-11-27 at 9 45 06 AM](https://github.com/beyondcode/herd-community/assets/6439071/0c046254-3617-4f05-aea8-4f753e0e6cf9)
![Screenshot 2023-11-27 at 9 37 04 AM](https://github.com/beyondcode/herd-community/assets/6439071/cef344c6-655e-470f-aa92-2d251614d11f)

However I can still access all my sites. Is this a known issue?

Restarting won't fix it. This is a fresh installation on Apple Silicon Mac. No prior valet installation or php installed using brew.

**Comment by mpociot** [comment]
This usually means that there are some stale processes that still access the old sock file. 
You can kill those processes: https://herd.laravel.com/docs/1/troubleshooting/restarting-services

**Reply by miagg** [reply]
and of course that did the trick! 🙏

**Comment by HasanAneek** [comment]
![Screenshot 2024-02-07 at 6 19 06 PM](https://github.com/beyondcode/herd-community/assets/82103572/0888c884-51af-45bf-aec4-148660671bae)
Why my FPM not showing and it shows mails which is locked, instead of FPM?

**Reply by kanjuo** [reply]
Did you end up resolving this? I am experiencing a similar issue

**Reply by kanjuo** [reply]
Well in my case this turned out to be the fact that I had not installed any Php version, so going to PHP > Install a version resolved it

**Comment by JeffreyDavidson** [comment]
@mpociot I'm having this problem to however my log is saying this. 

![CleanShot 2025-05-01 at 13 33 55@2x](https://github.com/user-attachments/assets/67e4c19e-78ea-4ca4-9ab2-b7554f5f302a)
